web/src/main/webapp/includes/global_functions_javascript.js

```diff
@@ -1,3 +1,22 @@
+function checkDataStatus() {
+
+        objImage=document.getElementById('status_top');
+        if (objImage != null && objImage.src.indexOf('images/icon_UnsavedData.gif')>0) {
+            return confirm('<fmt:message key="you_have_unsaved_data" bundle="${resword}"/>');
+        }
+
+        return true;
+    }

```

Comment:
It looks like viewSectionDataEntry.jsp had commented out checkDataStatus function. These changes will bring back the checkDataStatus function for that jsp. We should verify why that function was commented out and whether it's ok to bring it back.

Reply:
There are some jsps that have this javascript file imported and also have these functions defined within the jsp file. We should make sure that these functions don't interfere with each other. E.g. administrativeEditing.jsp has this scenario.